docs/upgrade.txt

```diff
@@ -118,8 +118,8 @@ problematic:
   original BSON type.
 
 - Numerically-indexed PHP arrays would be serialized as BSON documents if there
-  was a gap in their key sequence. Such gaps were easily caused by unsetting a
-  key to remove an element and forgetting to numerically reindex the array.
+  was a gap in their key sequence. Such gaps were caused by unsetting a key to
```

Comment:
No objection to removing "easily" here, as the sentence is just as clear without it, but I'd argue the word match here is a false positive. This language wasn't assuming that something should be "easy" for users to understand or do; rather, it was stating that a common (and valid) operation in PHP could produce an unintentional outcome in BSON serialization.

Reply:
I agree 👍🏻